### ISSUE TITLE
Generate Trajectories from ROM, Desired Velocity, and Reps Data 

### ISSUE THREAD

**Issue by achung123** [achung123/ROM_Rehab_Control#2]

**Comment by achung123** [comment]
Currently looking into Minimum-Jerk trajectory generation. Issue is we want to follow a specified path that is going to be represented as a discretized version of the actual continuous path. Most algorithms cover PTP motion so might have to use some kind of interpolation.

**Comment by chenxinencode** [comment]
Can you update the relevant paper or docs?I find your code is very helpful and want to read it .

**Comment by achung123** [comment]
@chenxinencode , sure! I can give a small update here. What part of the code base are you interested in? Also while I wait for your response I will hunt down the papers/textbooks I looked into for this project.

**Comment by chenxinencode** [comment]
Yes,I'm very interested,I am in the project of snake robot's path following,I have encountered the problem of convert the Sliding Mode Controller into code,So I want to find some code for reference.If you can update your code's relavent docs,I can better understand it.Thanks!
